entries/change.xml

```diff
@@ -52,6 +52,9 @@ $( "#other" ).click(function() {
     </code></pre>
     <p>After this code executes, clicks on <samp>Trigger the handler</samp> will also alert the message. The message will display twice, because the handler has been bound to the <code>change</code> event on both of the form elements.</p>
     <p>As of jQuery 1.4, the <code>change</code> event bubbles in Internet Explorer, behaving consistently with the event in other modern browsers.</p>
+    <div class="warning">
+      <p><strong>Note: </strong> dynamically changing the value of an input element, using <a href="/val"><code>.val()</code></a> for example, won't fire the event.</p>
```

Comment:
The word "dynamically" is a little bit ambiguous here, since someone might interpret a user-generated change to be dynamic. Perhaps "Changing the value property of an input element using JavaScript..." instead?